packages/backend/config/parking_config.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
**issue (blocking):** This is awesome work. However, I think that the database details should be dynamic and unique to each users machine since we do not have any remote database service for postgres

Reply:
reduced and fixed issue